src/DiffSharp.Tests/TestDerivatives.fs

```diff
@@ -3620,6 +3620,8 @@ type TestDerivatives () =
     [<Test>]
     member this.TestDerivativeConv3Ds2p2 () =
       for combo in Combos.AllDevicesAndBackends do
+       // 3D dilations not working correctly in LibTorch
```

Comment:
We need to fix this. In the meantime, do we raise an exception to the user when a 3d dilation with libtorch is requested?

Reply:
I mean, we need to ensure that it is not silently giving incorrect results without any error. Just disabling the test for libtorch is not great because we can forget about fixing the underlying problem in the libtorch backend. 3d convolutions are important to support.

Reply:
Yes, it needs to be fixed - the problem is the Dilate operation as I mentioned.  I think we can do it in a separate PR, or else maybe you could do it in terms of Scatter?

It does raise an error, see here

https://github.com/DiffSharp/DiffSharp/pull/106/files#diff-dca951cab2ee0dc4d6b2ad884ac2e1d8R276

Reply:
Ok sounds good. I can look at implementing that in libtorch backend with scatter, while also adding scatter to DiffSharp. I'm opening an issue to remember this. #116

Reply:
Cool, I'll be very glad when that Dilate code goes :-)